shell/run_tests.sh

```diff
@@ -155,32 +181,22 @@ sed -i "s#CDASH_URL#${CDASH_URL}#g"           "${JEDI_BUNDLE_DIR}/CTestConfig.cm
 sed -i "s#CDASH_URL#${CDASH_URL}#g"           "${JEDI_BUNDLE_DIR}/CTestConfig.cmake"
 sed -i "s#TEST_TARGET_NAME#${TRIGGER_REPO}#g" "${JEDI_BUNDLE_DIR}/CTestConfig.cmake"
 
-# Update the CMakeLists.txt files to include cdash integration. Note that the test
-# uses two different CMakeLists.txt files for unit and integration tests (starting with
-# unit tests) and each file needs the cdash integration. Both of these cmake files
-# were prepared by the GitHub action test launcher.
-echo "include(cmake/cdash-integration.cmake)" >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
-echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
-echo "include(CTest)"                         >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
-echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
+# Update the CMakeLists.txt files to include cdash integration. Note that the test may
+# use two different CMakeLists.txt files for unit and integration tests (starting with
+# unit tests) and each file needs the cdash integration. These cmake files are
+# prepared by the GitHub action test launcher.
+if [ -f "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration" ]; then
+    echo "include(cmake/cdash-integration.cmake)" >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
+    echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
+    echo "include(CTest)"                         >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
+    echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt.integration"
+fi
 # Update the unittest CMakeLists.txt file to include cdash integration.
 echo "include(cmake/cdash-integration.cmake)" >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt"
 echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt"
 echo "include(CTest)"                         >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt"
 echo ""                                       >> "${JEDI_BUNDLE_DIR}/CMakeLists.txt"
 
-
-if [ $? -ne 0 ]; then
-    if grep -qi "remote: Invalid username or password." configure_1.log; then
-        util.check_run_fail $TRIGGER_REPO_FULL $UNIT_RUN_ID "Failure: see jcsda-internal/CI/issues/137"
-    else
-        util.check_run_fail $TRIGGER_REPO_FULL $UNIT_RUN_ID "Bundle configuration failed"
-    fi
-    util.check_run_skip $TRIGGER_REPO_FULL $INTEGRATION_RUN_ID
-    util.evaluate_debug_timer_then_cleanup
-    exit 0
-fi
```

Comment:
This block got duplicated into this location at some point during the big refactor. It was doing nothing since it was evaluating the return code of an echo.